planner/core/common_plans.go

```diff
@@ -282,6 +289,10 @@ func (e *Execute) getPhysicalPlan(ctx context.Context, sctx sessionctx.Context,
 		} else {
 			planCacheCounter.Inc()
 		}
+		err = e.addHitInfo(sctx, "1")
```

Comment:
Please oo not use magic strings in the code. You can pre-define them like this:
```go
const (
	hintOptOn  string = "1"
	hintOptOff string = "0"
)
```

and use it here like this:
```go
		err = e.addHitInfo(sctx, hintOptOn)
```